docs/content/en/contribute/general/github/_index.md

```diff
@@ -0,0 +1,276 @@
+       Click the icon at the right end of that box to copy that URL.
+
+3. Run the **git clone** command from the shell of a local directory
+    to clone the forked repository to a directory on your local machine,
+    pasting in the URl you saved in the previous step:
+
+    ```console
+    git clone https://github.com/<UserName>/lifecycle-toolkit
+    ```
+
+    or
+
+    ```console
+    git clone git@github.com:<UserName>/lifecycle-toolkit.git
+    ```
+
+    Where <*UserName*> is your GitHub username.
+    The lifecycle-toolkit directory is now available in the local directory.
+
+4. Associate your clone with `upstream`.
+   To do this, use the same string you used to clone your fork.
+
+   * Be sure that you are in the root folder of the project
+     and run *git status* to confirm that you are on the `main` branch.
+   * Type the following to associate `upstream` with your clone,
+     pasting in the string for the main repo that you copied above.:
+
+     ```console
+     git remote add upstream https://github.com/keptn/lifecycle-toolkit.git 
+     ```
+
+### Create a new branch and make your changes
+
+Now you have your upstream setup in your local machine.
+You need to create a local branch where you will make your changes.
+Be sure that your branch is based on and sync'ed with `main`,
+unless you intend to create a derivative PR:
+
+The following sequence of commands does that:
+
+```console
+git checkout main
+git pull upstream main
+git push origin main
+git checkout -b <my-new-branch>
+```
+
+Now you can make your changes, build them locally to check formatting,
+then create a PR to add these changes to the documentation set
+
+### Submitting new content through a pull request
+
+If you have forked and cloned the repository,
+you can modify the documentation or add new material
+by editing the markdown file(s) in the local clone of your fork
+and then submitting a *pull request (PR)*.
+
+Note that you can also modify the source using the GitHub editor.
+This is very useful when you want to fix a typo or make a small editorial change
+but, if you are doing significant writing,
+it is generally better to work on files in your local clone.
+
+1. Execute the following and check the output to ensure that your branch is set up correctly:
+
+   ```console
+   git status
+   ```
+
+1. Do the writing you want to do in your local branch, checking the formatted version at `localhost:1314`.
+
+1. When you have completed the writing you want to do, close all files in your branch and run `git status` to confirm
+   that it correctly reflects the files you have modified, added, and deleted.
+
+1. Add and commit your changes.  Here, we commit all modified files but you can specify individual files to the
+   `git add` command.
+The `git commit -s` command commits the files and signs that you are contributing this intellectual property to the
+Keptn project.
+
+   ```console
+   git add .
+   git commit -s
+   ```
```

Comment:
The flow seems awkward here, doesn't it?  And if I'm brand new to all of this, does it make sense?